scripts/vulnerabilities.sh

```diff
@@ -46,7 +46,7 @@ echo -n "Scanning" >&2
 for image in "${images[@]}"; do
   echo -n "."
   results=$(gcloud beta container images describe --show-package-vulnerability --format json --verbosity error "${image}")
-  status=$(echo "${results}" | jq -r '.discovery_summary.discovery[].discovery.analysisStatus')
+  status=$(echo "${results}" | jq -r '.discovery_summary.discovery[] | select(.noteName == "projects/goog-analysis/notes/PACKAGE_VULNERABILITY") | .discovery.analysisStatus')
```

Comment:
Do you know what is the meaning of the "COMPLIANCE" result?

Reply:
This is the first image that includes the `COMPLIANCE` result.  After a brief search, I think it refers to a bunch of security standards.

References: 
- https://cloud.google.com/security-command-center/docs/compliance-management

Interestingly, compute image vulnerability report doesn't seem to map to any compliance standard controls: https://cloud.google.com/security-command-center/docs/concepts-vulnerabilities-findings#compute-image-findings. I don't think that report is useful for us at this moment.